README.md

```diff
@@ -25,6 +25,68 @@ event scraper functions.
 
 ## Quick Start
```

Comment:
Can you move the Legistar quickstart / code to up here. I generally like the "general purpose utils" and code snippets near the top. Extended documentation (like this PR) can always be specifically linked to whereas the code snippets should be like the first thing someone sees.